### PR TITLE
Run poetry upgrade, fix docs build

### PR DESCRIPTION
**Pull request by SimonBiggs**

### CODE REVIEW

**Comment by sjswerdloff** [comment]
do you still want this PR or should we close it without merging?

**Comment by SimonBiggs** [comment]
> do you still want this PR or should we close it without merging?

Let's keep this one. My intention here was to bump streamlit so that it can be done within main instead of needing @rembishj to do it in his PR first.

We will need to solve the docs issue though before merging though.

**Comment by SimonBiggs** [comment]
> We will need to solve the docs issue though before merging though.

Cheers for the link you provided in the other PR. It seems the fix will be quite simple:

https://github.com/executablebooks/jupyter-book/issues/1137#issuecomment-743994864

File: pyproject.toml [hunk 1]
```diff
@@ -84,6 +84,7 @@ jupyter-book = {version = ">=0.8.3", optional = true}  # groups = ["dev", "docs"
 sphinx-argparse = {version = "*", optional = true}  # groups = ["dev", "docs"]
 sphinxcontrib-napoleon = {version = "*", optional = true}  # groups = ["dev", "docs"]
 sphinx-book-theme = {version = "*", optional = true}  # groups = ["dev", "docs", "doctests"]
+sphinxcontrib-bibtex = {version = "<2.0.0", optional = true}  # groups = ["dev", "docs", "doctests"]
```
Comment (SimonBiggs) [inline pyproject.toml:87]: Upgrade issue fixed. 🙂

**Review by sjswerdloff — APPROVED** [review]

**Pull request merged**